test/Makefile

```diff
@@ -39,3 +46,9 @@ coverage:
 coverage-html:
 	python3 -m coverage html
 	xdg-open htmlcov/index.html
+
+cluster:
```

Comment:
can we rename the target to test-cluster? and also rename the clean target to something else?